### ISSUE TITLE
Using same files for GS65 8RF

### ISSUE THREAD

**Issue by advanc3dUA** [ErrorErrorError/msi-gs65-8se-opencore-macos#12]
Hello,

I've tried to get into MAC world for the first time with your old guide for Clover but I stucked on Apple logo with looking like the same problem which was described in issues. I found in that issue thread that you highly recommend to OpenCore instead of Clover.

Now I am gonna try it. So, my question is: if I will follow [this](https://dortania.github.io/OpenCore-Install-Guide/prerequisites.html) OpenCore guide, can I simply use all of your files in this repo instead of searching required one's for my GS65 8RF? By the way, I have already replaced original WLAN with BCM94352Z as you suggested in Clover thread. I checked that there is no big difference between 8SE & my 8RF but I still doubt it anyway.

Thanks for your reply.

**Comment by erikbdev** [comment]
I would much recommend the guide from dortania since I haven't updated my efi with the latest version. I'd also recommend to not just copy my efi but create your own and use mine as reference since there have been many things changed.

**Comment by advanc3dUA** [comment]
I did it! 

I downloaded your release and used your files to create my EFI folder based on latest release of OC. After that I checked every kext and driver you have if there were any newer version. If I was able to find update - used it. Left your config untouched (tried to create my own but was unable to get to install). And it looks like everything except trackpad works (& webcam but I don't need it) fine. Even sound, WLAN, battery indicator etc.

Before asking for the help with trackpad I decided to mount hidden EFI drive of my MAC OS SSD (copied my EFI folder from flashdrive to boot without it) and used yours VoodooI2C, VoodooI2CHID and VoodooPS2Controller kexts but it didn't helped (if I understand right, those kexts responsible for trackpad work). I am very new to MAC OS - maybe there is a simple thing to turn on to get it to work or you have any other ideas why it doesn't work?

Thank you for promt reply.

**Comment by advanc3dUA** [comment]
I have tried a lot of different file combinations - didn't matter. For some reason I decided to check config file and I have found that trackpad kext was turned off. Switched it to "true" and... fixed!

All thanks for help goes to me :).

**Comment by erikbdev** [comment]
> I have tried a lot of different file combinations - didn't matter. For some reason I decided to check config file and I have found that trackpad kext was turned off. Switched it to "true" and... fixed!
> 
> 
> 
> All thanks for help goes to me :).

If you're talking about VoodooInpur.kaxt then only one should be on. Usually VodooPS2Controller and voodooI2C has one, this you only lees one.

**Comment by advanc3dUA** [comment]
Nope, I found their VoodooPS2Trackpad.kext with such screaming name and turned it on. This helped :).

**Comment by advanc3dUA** [comment]
Btw, is there any chance to get mDP to work? I've tried -igfxtypec boot-arg and got success with monitor connected via USB-C to DP cable. This cable has not enough bandwith for my monitor (I have 144Hz 2560x1440 screen) and can work only in 60Hz mode + I have to switch it everytime from mDP-DP cable (monitor has only one DP connection). 

My mDP-DP cable is fast enough and I use it for my Windows system. I prefer using this one for MacOS too, so I won't need any cable switching. 

Can you please help with it?

**Comment by erikbdev** [comment]
> Btw, is there any chance to get mDP to work? I've tried -igfxtypec boot-arg and got success with monitor connected via USB-C to DP cable. This cable has not enough bandwith for my monitor (I have 144Hz 2560x1440 screen) and can work only in 60Hz mode + I have to switch it everytime from mDP-DP cable (monitor has only one DP connection). 
> 
> 
> 
> My mDP-DP cable is fast enough and I use it for my Windows system. I prefer using this one for MacOS too, so I won't need any cable switching. 
> 
> 
> 
> Can you please help with it?

You have to go to windows and see if mDP is connected to the nVidia card. If it is then you cannot use it on macOS

**Comment by sawyerfeng** [comment]
@fash1k My laptop is MSI GS65 Thin 8RF(16Q2) gtx1070maxq, I tried to update opencore, I tried the 6.6 and 6.9version,and use @ErrorErrorError files, but both of them are no use, I wonder how you have done this job. I would be grateful if you could share me your experiences,  and if you open source  your efi, I would be much more appreciated.![20210512_005220.jpg](https://user-images.githubusercontent.com/52705605/117855741-5afc9d80-b2bd-11eb-89e6-bcf60cae79fb.jpg)
